misc/formatter/modules/formatter-cli/src/main/java/org/ballerinalang/formatter/cli/FormatUtil.java

```diff
@@ -188,9 +194,12 @@ static void execute(List<String> argList, boolean helpFlag, String moduleName, S
                 }
             } else {
                 BuildProject project;
-
+                FormattingOptions options;
                 try {
                     project = BuildProject.load(sourceRootPath, constructBuildOptions());
+                    options = FormattingOptions.builder()
+                            .build(project.sourceRoot(),
+                                    FormatterUtils.loadFormatSection(project.currentPackage().manifest()));
```

Comment:
Seems to be a repetition of L131 - L133? Let's extract this logic out to a method.